src/quo/components/wallet/confirmation_progress/component_spec.cljs

```diff
@@ -0,0 +1,99 @@
+(ns quo.components.wallet.confirmation-progress.component-spec
+  (:require [quo.core :as quo]
+            [reagent.core :as reagent]
+            [test-helpers.component :as h]))
+
+(defn- get-test-data
+  [{:keys [state network]
+    :or   {state :pending network :mainnet}}]
+  {:counter             (reagent/atom 0)
+   :total-box           85
+   :progress-value      "10"
+   :network             network
+   :state               state
+   :customization-color :blue})
+
```

Comment:
@mmilad75, could you please also add the test for `[quo/confirmation-propgress]` without props?
p.s. also there is a misprint in `quo/confirmation-propgress` name)